src/TimeGrid.js

```diff
@@ -348,7 +352,7 @@ export default class TimeGrid extends Component {
     let gutterCells = this._gutters;
 
     if (!width) {
-      width = Math.max(...gutterCells.map(getWidth));
+      width = Math.max(...gutterCells.map(getWidth).filter(Boolean));
```

Comment:
this will give weird results when there are no truthy results. e.g `Math.max(...[0,0,0].filter(Boolean)) // -Infinity` which in javascript land is truthy, and would set the gutterWidth to `-Infinity`